lib/health_check/middleware_health_check.rb

```diff
@@ -0,0 +1,42 @@
+module HealthCheck
+  class MiddlewareHealthcheck
+
+    def initialize(app)
+      @app = app
+    end
+
+    def call(env)
+      uri = env['PATH_INFO'.freeze]
```

Comment:
don't think it's actually needed here anyways, it just freaks out devs what might be going here